Resources/Prototypes/SS220/Entities/Clothing/Sponsor/clothing.yml

```
@@ -0,0 +1,281 @@
+    storedRotation: -90
+
+- type: entity
+  parent: ClothingSponsorEyesBase
+  id: ClothingSponsorWaltuhGlasses
+  name: прозрачные авиаторы
+  description: Простые дешёвые очки с корректирующими линзами
+  components:
+  - type: Sprite
+    sprite: SS220/Clothing/Sponsor/waltuh/glasses.rsi
+  - type: Clothing
+    sprite: SS220/Clothing/Sponsor/waltuh/glasses.rsi
+  - type: VisionCorrection
+
+- type: entity
+  parent: ClothingSponsorUniformBase
+  id: ClothingSponsorWaltuhInnerShirt
+  name: бледная рубашка
+  description: Вы - самый обычный семьянин
+  components:
+  - type: Sprite
+    sprite: SS220/Clothing/Sponsor/waltuh/innershirt.rsi
+  - type: Clothing
+    sprite: SS220/Clothing/Sponsor/waltuh/innershirt.rsi
+
+- type: entity
+  parent: ClothingSponsorOuterStorageBase
+  id: ClothingSponsorWaltuhOuterShirt
+  name: плотная чёрная рубашка
+  description: Отдаёт химией.
+  components:
+  - type: Sprite
+    sprite: SS220/Clothing/Sponsor/waltuh/outershirt.rsi
+  - type: Clothing
+    sprite: SS220/Clothing/Sponsor/waltuh/outershirt.rsi
+
+- type: entity
+  parent: ClothingSponsorUniformBase
+  id: ClothingSponsorGoldClownSuit
+  name: золотой костюм клоуна
+  description: Ууу, блестяшка!
+  components:
+  - type: Sprite
+    sprite: SS220/Clothing/Sponsor/goldclown/goldclownsuit.rsi
+  - type: Clothing
+    sprite: SS220/Clothing/Sponsor/goldclown/goldclownsuit.rsi
+
+- type: entity
+  parent: ClothingSponsorShoesBase
+  id: ClothingSponsorGoldClownShoes
+  name: золотые клоунские туфли
+  description: Шааааайни!
+  components:
+  - type: Sprite
+    sprite: SS220/Clothing/Sponsor/goldclown/goldclownboots.rsi
+  - type: Clothing
+    sprite: SS220/Clothing/Sponsor/goldclown/goldclownboots.rsi
+  - type: FootstepModifier
+    footstepSoundCollection:
+      collection: FootstepClown
+
+- type: entity
+  parent: ClothingSponsorMaskBase
+  id: ClothingSponsorGoldClownMask
+  name: золотая маска клоуна
+  description: Выглядит очень тяжелой
+  components:
+  - type: Sprite
+    sprite: SS220/Clothing/Sponsor/goldclown/goldclownmask.rsi
+  - type: Clothing
+    sprite: SS220/Clothing/Sponsor/goldclown/goldclownmask.rsi
+  - type: BreathMask
+  - type: IdentityBlocker
+
+- type: entity
+  parent: ClothingSponsorOuterStorageBase
+  id: ClothingSponsorDante
+  name: вишнёвый плащ
+  description: Але?
+  components:
+  - type: Sprite
+    sprite: SS220/Clothing/Sponsor/dante.rsi
+  - type: Clothing
+    sprite: SS220/Clothing/Sponsor/dante.rsi
+
+- type: entity
+  parent: ClothingSponsorMaskBase
+  id: ClothingSponsorDallas
+  name: демократическая маска
+  description: Вошли и вышли, дело пяти минут.
+  components:
+  - type: Sprite
+    sprite: SS220/Clothing/Sponsor/dallas/mask.rsi
+  - type: Clothing
+    sprite: SS220/Clothing/Sponsor/dallas/mask.rsi
+  - type: BreathMask
+  - type: IdentityBlocker
+
+- type: entity
+  parent: ClothingSponsorHeadBase
+  id: ClothingSponsorMafiaHat
+  name: розовая шляпа
+  description: Я - Зубен...
```

```suggestion
  description: Я - Мафиозник!
```
мне кажется, так лучше впишется, потому что "Зубен" - как то стрёмно (да, я понимаю, что фраза обрывается), и как-то ООС